src/sentry/plugins/sentry_mail/models.py

```diff
@@ -23,6 +23,7 @@
 from sentry.plugins import register
 from sentry.plugins.base.structs import Notification
 from sentry.plugins.bases.notify import NotificationPlugin
+from sentry.tasks.unmerge import get_environment_name
```

Comment:
@tkaemming Is there a better way to derive this? I don't like pulling this random function out of `sentry.tasks.unmerge`.

Reply:
There's not really a clean way to do it — the options I can think of are either importing it here or duplicating the logic (it's not complex.)

I'd considered adding it as a property on the `Event` model but since that requires the `data` field to be populated, the possibility of enabling someone to accidentally implicitly fire off N serial queries in a loop made me nervous, so I didn't.